src/main/java/com/minecolonies/core/commands/EntryPoint.java

```diff
@@ -92,7 +93,7 @@ public static void register(final CommandDispatcher<CommandSourceStack> dispatch
           .addNode(new CommandBackup().build())
           .addNode(new CommandResetPlayerSupplies().build())
           .addNode(new CommandHelp().build())
-          .addNode(new ScanCommand().build())
+                                               .addNode(ScanCommand.build())
           .addNode(new CommandPruneWorld().build());
```

Comment:
We should probably adjust the formatter to not make those huge gaps, makes reading harder =D

Reply:
ye formatter is being stupid :D